dirsrvtests/tests/suites/fourwaymmr/fourwaymmr_test.py

```diff
@@ -434,7 +434,7 @@ def list_agmt_towards(topo_m4, serverid):
     res = []
     for inst in topo_m4:
         for agmt in Agreements(inst).list():
-            if agmt.get_attr_val_utf8(AGMT_PORT) == topo_m4.ms[serverid].port:
+            if int(agmt.get_attr_val_utf8(AGMT_PORT)) == topo_m4.ms[serverid].port:
```

Comment:
I think you should be able to use 'get_attr_val_int()' here.
Hence making it: `if agmt.get_attr_val_int(AGMT_PORT) == topo_m4.ms[serverid].port:`

Reply:
Thanks @droideck for the tip (somehow I feel a bit stupid for this one :) ) !

Reply:
No problem :)
We just don't have good discoverability for some things, so that it may be confusing sometimes.